src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -244,16 +280,35 @@ void RadioSpinel<InterfaceType>::Init(bool aResetRadio, bool aSkipRcpCompatibili
     SuccessOrDie(error);
 }
 
+template <typename InterfaceType>
+void RadioSpinel<InterfaceType>::SetCallbacks(const struct RadioSpinelCallbacks &aCallbacks)
+{
+#if OPENTHREAD_CONFIG_DIAG_ENABLE
+    assert(aCallbacks.mDiagReceiveDone != nullptr);
+    assert(aCallbacks.mDiagTransmitDone != nullptr);
+#endif
+    assert(aCallbacks.mEnergyScanDone != nullptr);
+    assert(aCallbacks.mReceiveDone != nullptr);
+    assert(aCallbacks.mTransmitDone != nullptr);
+    assert(aCallbacks.mTxStarted != nullptr);
+
+    mCallbacks = aCallbacks;
+}
+
 template <typename InterfaceType> void RadioSpinel<InterfaceType>::ResetRcp(bool aResetRadio)
 {
     bool hardwareReset;
     bool resetDone = false;
 
-    mIsReady    = false;
+    // Avoid resetting the device twice in a row in RCP_FT architecture
```

Comment:
What is RCP_FT stands for?

Reply:
Nice catch;) This should be removed before creating this PR.  FT is an abbreviation of our team name. RCP_FT was used to distinguish multipan RCP from regular(singlepan) one. Will update this. Should be " in Multipan RCP architecture"